src/cpp/transport/udp/UDPv4AgentLinux.cpp

```diff
@@ -67,6 +69,24 @@ UDPv4Agent::~UDPv4Agent()
 
 bool UDPv4Agent::init()
 {
+    messagesFromFlightController = &(prometheus::BuildCounter()
+        .Name("messages_from_flightcontroller_count")
+        .Help("Number of messages sent towards Mission Computer")
+        .Register(*metrics_registry).Add({}));
+
+    messagesToFlightController = &(prometheus::BuildCounter()
+        .Name("messages_to_flightcontroller_count")
+        .Help("Number of messages sent towards Flight Controller")
+        .Register(*metrics_registry).Add({}));
+
+    auto metrics_port = getenv("METRICS_PORT");
+    if (metrics_port != nullptr) // start HTTP endpoint only if requested
```

Comment:
This condition makes it so that if ENV var is not set, the metrics HTTP server is not started

So it can be disabled if needed

Reply:
Nice idea!